_data/retail.yml

```diff
@@ -25,7 +25,6 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
-      lang: pl
```

Comment:
Actually, why did you remove the lang attribute? Does Allegro support English? If so, could you provide a link?
I could only find English version of the API but it's not enough.